packages/yoga/src/Input/web/Input.test.jsx

```diff
@@ -204,7 +204,42 @@ describe('<Input />', () => {
         </ThemeProvider>,
       );
 
+      debug();
```

Comment:
❓ Doubt | @nataliosilveiraext 

I think that you forgot this `debug` here.

Reply:
was forgotten, I will remove it